### ISSUE TITLE
how to limit number of posts in the atom feed ?

### ISSUE THREAD

**Issue by arpinux** [venthur/blag#112]
i would like to fix the atom feed to a certain number. is there a way to perform it ?

**Comment by venthur** [comment]
Not really, but one could think of something using a new setting in the config file that defaults to "no limit" if not existing. Do you have issues with the the current unlimited number of posts in the feed?

**Comment by arpinux** [comment]
hi, thx for your quick answer and for blag :)

no issue at all. in fact, i'm upgrading my blog from Pluxml to blag and the feed is full of my +100 posts :D 
i just wanted to preserve people from receiving all those notifications ;)

an option in the config file to limit number of post would be great but if it's not possible, i will edit the atom.xml before uploading on my server :)

**Comment by venthur** [comment]
Let me know how the migration happened :) good luck!

**Comment by arpinux** [comment]
[done](https://blog.arpinux.org), thx again for your blag :)
